src/bot/arena_bot.py

```diff
@@ -1,15 +1,26 @@
 import aiohttp
 from discord.ext.commands import Bot
+from glob import glob
+from pathlib import Path
 
 
 class ArenaBot(Bot):
     def __init__(self, command_prefix, *args, **kwargs):
         super().__init__(command_prefix, *args, **kwargs)
-
         self.session = aiohttp.ClientSession()
 
     async def on_ready(self):
         print(f"Logged in as {self.user}")
-    
+
     async def on_disconnect(self):
         await self.session.close()
+
+    @classmethod
+    def create(cls) -> 'ArenaBot':
```

Comment:
Can we use
```py
from __future__ import annotations
```
to allow annotating returns before class definition?

Reply:
Sure, I'll take a look at how to do that.